docs/API.md

```diff
@@ -377,6 +377,24 @@ const tracer = require('dd-trace').init({
 })
 ```
 
+<h3 id="span-tag">Span Tags</h3>
+The Datadog UI uses tags to set span level metadata. Custom tags may be set by grabbing the active span from the global tracer and setting a tag with `setTag` method.
+
+For example:
+```javascript
+app.get('/shopping_cart/:customer_id', (req, res) => {
+  // Get the active span
+  const span = tracer.scope().active()
+  if (span !== null) {
+    // customer_id -> 254889
+    span.setTag('customer.id', req.params.customer_id)
```

Comment:
I think this pattern was proliferated by another team. How do folks feel about using something more modern and terse, removing the whole `if` statement?

```js
span?.setTag('customer.id', req.params.customer_id)
```

OTOH I suppose the verbosity of the `if` statement explicitly tells the developer to watch out for a span value of `null`...

Reply:
The more I think about it the more I realize the explicit null check is good.

Reply:
I would generally go for the if-block form just because the `?.` syntax is a bit mysterious to many still. Though _personally_ I would just do `if (span)` rather than `if (span !== null)` to make it simpler to read.